### ISSUE TITLE
Dark mode option

### ISSUE THREAD

**Issue by adrianriyadi** [kakawait/hugo-tranquilpeak-theme#384]
is it possible to make the dark mode for night reader? :)

**Comment by kakawait** [comment]
@adrianriyadi good idea but really lack of time to prioritize that feature request for short term.

**Comment by AnceretMatthieu** [comment]
I implemented this feature in my fork. I hope I can do a PR to integrate this (https://github.com/AnceretMatthieu/hugo-tranquilpeak-theme/pull/1). I still have a bit of work to make my edit clean.

**Comment by kakawait** [comment]
> I implemented this feature in my fork. I hope I can do a PR to integrate this ([AnceretMatthieu#1](https://github.com/AnceretMatthieu/hugo-tranquilpeak-theme/pull/1)). I still have a bit of work to make my edit clean.

Sorry for late (really late) response. But yes @AnceretMatthieu I'll be glad to accept your contribution.

I saw the result on https://anceret-matthieu.fr/ is really clean 👍

**Comment by palladius** [comment]
If this is integrated, I don't understand how I can integrate it in my site.
I agree the functionality in https://anceret-matthieu.fr/ is pretty slick! I love it!
